core/src/commonMain/kotlin/com/xebia/functional/xef/auto/AI.kt

```diff
@@ -145,23 +136,16 @@ class AIScope(
    * within [block] will be finalized after [block] finishes.
    */
```

Comment:
We should update this piece of documentation by removing the `resourceScope` reference.
```
/**
   * Creates a new scoped [VectorStore] using [store], which is scoped to the [block] lambda. The
   * [block] also runs on a _nested_ [resourceScope], meaning that all additional resources created
   * within [block] will be finalized after [block] finishes.
   */
```